forgero-core/src/main/java/com/sigmundgranaas/forgero/core/property/v2/feature/PropertyData.java

```diff
@@ -40,4 +40,8 @@ public class PropertyData implements Property {
 	public String type() {
 		return type;
 	}
+
+	public boolean is(String type) {
+		return this.type.equals(type);
+	}
```

Comment:
Small nitpick here, I'd argue that the method names would be more clear if it were `getType()` and `isOfType(String type)`.

Reply:
I agree with the last one, it makes it more clear. For the get/set convention, I dunno, I am  quite tired of writing get in front of everything xD

Reply:
Haha fair enough, though get/set in front of the method names is more consistent with existing code.

Reply:
Yeah that's true, but in general I am moving away from the getter setter syntax for new code. But there are definitely places here in this PR where getters make the functions more readable